docs/api/createI13nNode.md

```diff
@@ -29,6 +32,30 @@ var I13nAnchor = createI13nNode('a', {
 </I13nAnchor>
 ```
 
+If you have cases that you cannot replace the links with `I13n Component`, e.g., you are using `dangerouslySetInnerHTML`. You can integrate the `scanLinks` options to get links tracked, we will not suggest to use this since it happens on client side and create additional DOM manipulation. 
```

Comment:
If you have a use case where you cannot replace the links with `I13n Component`, i.e, when using `dangerouslySetInnerHTML`. You can integrate the `scanLinks` options to automatically track these links. This option should be used sparingly since it can create additional DOM manipulations.